src/rete/controls/LabeledInputControl.tsx

```diff
@@ -7,10 +7,28 @@ export class LabeledInputControl extends Classic.InputControl<'number'> {
         change?: () => void,
         readonly?: boolean
     ) {
-        super('number', { initial: value, readonly: Boolean(readonly), change })
+
+        super('number', { initial: value, readonly: Boolean(readonly), change: change })
+    }
+
+    setValue(newValue: number): void {
+        let incrementor = 1
+        if (this.label === 'Base Gain') {
```

Comment:
Let's aim for something more flexible than checking the label - we're going to end up wanting to use a field with customizable increments in way more places than just gain and frequency. Ideally, in the constructor for a node, I'd be able initialize an input control with `new LabeledInputControl(initial: 1234, label: 'whatever', readonly: false, change: null, increment: 0.1)` or something like that.